src/kademlia/get_item.cpp

```diff
@@ -70,6 +70,15 @@ void get_item::got_data(bdecode_node const& v,
 		{
 			if (!m_data.assign(v, salt, seq, pk, sig))
 				return;
+
+			if (!m_put_requested)
```

Comment:
If I'm a user, when I call put_item, I will suspect only be notified when "put" is done. 
User can do nothing with the non-authoritative date - he need the authoritative "seq". And more, since user just call put_item, he shouldn't be disturbed by other event except the put_complete. 
BTW, Right now the put_item alert is not implemented correctly - user receive notification when he get those nodes can accept the "put". This is another issue and I wouldn't address it in this request.